src/Tooltip/Tooltip.styles.ts

```diff
@@ -0,0 +1,88 @@
+import styled, { css } from 'styled-components'
```

Comment:
don't have strong personal preferences but we tend to keep all styled component at the bottom of the actual `tsx` file. Considering that this is a public library might be good to keep the same standards

Reply:
Yep no worries! I showed it to Rowan as I knew it might be a bit strange moving having a seperate `styles.ts` file. Have moved them underneath the component now 😀